packages/auth/src/index.ts

```diff
@@ -37,3 +38,10 @@ export const {
     },
   },
 });
+
+export { GET, POST, signIn, signOut };
+/**
+ * This is the main way to get session data for your RSCs.
+ * This will de-duplicate all calls to next-auth's default `auth()` function and only call it once per request across all components
+ */
+export const auth = cache(defaultAuth);
```

Comment:
This doesn't work in middleware does it?

Reply:
In Next's middleware.ts?

Reply:
Then I believe we could add a comment somewhere, or in Readme.md explaining to also export `defaultAuth`  for that use case. What do you suggest?

Reply:
i think we should have a middleware.ts file by default tbh

Reply:
¯\_(ツ)_/¯ in this case yea sure

Just thought that this repo was supposed to be a "turbo" version of ct3a. And since in there we don't specify middleware files, don't see why we should also do it here? Could be a good idea though.

Reply:
afaik middleware is the only way to make the session refresh?